src/compile.coffee

```diff
@@ -146,7 +148,7 @@ compileOriginExampleName = (mediaType, httpResponseElement, exampleNo) ->
     if exampleNo
       exampleName = "Example #{exampleNo}"
   else
-    statusCode = httpResponseElement.statusCode.toValue()
+    statusCode = httpResponseElement.statusCode?.toValue() or '200'
```

Comment:
Maybe `200` shouldn't be the default in some cases. If there is no schema or body then `204` may make more sense?

Reply:
Empty bodies could be 205 as well... I'm not sure I want to dig into more complicated logic at this moment. No schema/body can just mean there are no assertions. Let's see how users work with this and then let's reiterate on what we can do better.